### DISCUSSION TITLE
List of organisations using JSON Schema

### DISCUSSION THREAD

**Discussion by Bhupesh-V** [json-schema-org/community#227] [Q&A] [answered]
Hey I recently learned about JSON schema (I love it).
Was wondering wether we could have a public list of organisations that use JSON schema (atleast in their public projects)?

If such list doesn't exist, would it be a good idea to scrape github dependencies to get that list. Just doing a bit browsing reveals that these big organisations (& projects) use JSON schema (the actual count will be bigger than this):
1. [salesforce](https://github.com/salesforce/CodeRL)
2. [Python poetry](https://github.com/python-poetry/poetry)
3. [canonical](https://github.com/canonical/bundle-kubeflow)
4. [microsoft](https://github.com/microsoft/datashaper)

> I belive listing our these names (& gathering more data) might help people gain more trust in the project & hence be a part of community

Looking forward to know community thoughts 👀

**Comment by handrews** [comment]
We do have some case studies on [the blog](https://json-schema.org/blog) but I'm not aware of a more comprehensive list.

It would be good to list both direct usage and downstream projects (e.g. OpenAPI, AsyncAPI, Web of Things) that build on JSON Schema.

When it comes to corporate usage, I'm unclear on whether there are legal concerns about such a listing implying endorsement.  I believe we get permission for each case study.

**Comment by jviotti** [comment, marked as answer]
@Bhupesh-V @handrews I actually maintain a list of companies that mention they use JSON Schema in some way or another. Check out the "Adoption" section of my Awesome JSON Schema project: https://github.com/jviotti/awesome-jsonschema#adoption

**Reply by Bhupesh-V** [reply]
Hey this looks great. Thanks for sharing.
Is it community maintained or did you get the data across different SDKs uses across github? I am assuming actual count will be way higher right?

**Reply by jviotti** [reply]
@Bhupesh-V It is community maintaining and collected through a lot of Googling :) If there is anything we missed, feel free to create a PR or an issue!
